spec/models/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let!(:user) { create(:user) }
+  let!(:challenge_id) { create(:challenge).id }
+  let!(:solution) { create(:solution, user_id: user.id, challenge_id: challenge_id)}
+
+  context 'associations' do
+    subject { Comment.new(user_id: user.id, solution_id: solution.id) }
```

Comment:
Based on the previous comment, you don't need to define a explicit `subject`. rspec will help you to define the implicit subject `(Comment.new)`